File: claude-experiments/ai-lang/benchmarks/rust/src/bin/fib.rs
// Naive recursive Fibonacci — mirrors benchmarks/ail/fib.ail.

fn fib(n: i64) -> i64 {
    if n < 2 { n } else { fib(n - 1) + fib(n - 2) }
}

fn main() {
    let n = 32;
    let t0 = std::time::Instant::now();
    let r = fib(n);
    let ms = t0.elapsed().as_millis();
    println!("RESULT fib {} ms checksum={}", ms, r);
}

File: claude-experiments/oo-lang/bench/fib.py
import time
def fib(n): return n if n < 2 else fib(n-1) + fib(n-2)
t0 = time.time(); r = fib(32); t1 = time.time()
print(f"fib(32) = {r}  [{(t1-t0)*1000:.1f} ms]")
